### PR TITLE
Add release notes for Trino Gateway 5

### PR DESCRIPTION
**Pull request by mosabua**
## Description

Assemble the release notes for Trino Gateway 5 release.

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

## Verification for each pull request

Format: maintainer, PR/issue number, ✅ / ❌ rn ✅ / ❌ docs
✅ rn - release note added and verified, or assessed to be not necessary, set to ❌ rn before completion
✅ docs - need for docs assessed and merged, or assessed to be not necessary, set to ❌ docs before completion

## 6 Dec 2023

* #108 ✅ rn ✅ docs

## 7 Dec 2023

* #93 ✅ rn ✅ docs 

## 8 Dec 2023

* #104 ✅ rn ✅ docs
* #105 ✅ rn ✅ docs
 
## 15 Dec 2023

* #110 ✅ rn ✅ docs

## 19 Dec 2023

* #112 ✅ rn ✅ docs

## 20 Dec 2023

* #111 ✅ rn ✅ docs

## 21 Dec 2023 

* #106 ✅ rn ✅ docs
* #115 ✅ rn ✅ docs

## 29 Dec 2023

* #118 ✅ rn ✅ docs

## 30 Dec 2023 

* #121 ✅ rn ✅ docs
* #124 ✅ rn ✅ docs
* #122 ✅ rn ✅ docs
* #120 ✅ rn ✅ docs

## 31 Dec 2023 

* #126 ✅ rn ✅ docs
* #127 ✅ rn ✅ docs
* #128 ✅ rn ✅ docs
* #129 ✅ rn ✅ docs
* #130 ✅ rn ✅ docs

## 2 Jan 2024

* #133 ✅ rn ✅ docs
* #136 ✅ rn ✅ docs
* #135 ✅ rn ✅ docs
* #134 ✅ rn ✅ docs
* #137 ✅ rn ✅ docs

## 3 Jan 2024

* #138 ✅ rn ✅ docs
* #143 ✅ rn ✅ docs

## 5 Jan 2024

* #139 ✅ rn ✅ docs
* #141 ✅ rn ✅ docs
* #148 ✅ rn ✅ docs
* #140 ✅ rn ✅ docs
* #151 ✅ rn ✅ docs

## 7 Jan 2024

* #154 ✅ rn ✅ docs

## 8 Jan 2024

* #99 ✅ rn ✅ docs
* #150 ✅ rn ✅ docs
* #152 ✅ rn ✅ docs
* #156 ✅ rn ✅ docs
* #144 ✅ rn ✅ docs

## 10 Jan 2024

* #157 ✅ rn ✅ docs
* #158 ✅ rn ✅ docs
* #142 ✅ rn ✅ docs

## 11 Jan 2024

* #146 ✅ rn ✅ docs
* #160 ✅ rn ✅ docs

## 17 Jan 2024

* #162 ✅ rn ✅ docs

## 18 Jan 2024

* #96 ✅ rn ✅ docs
* #167 ✅ rn ✅ docs
* #97 ✅ rn ✅ docs

## 19 Jan 2024

* #169 ✅ rn ✅ docs
* #171 ✅ rn ✅ docs

## 20 Jan 2024

* #173 ✅ rn ✅ docs
* #175 ✅ rn ✅ docs
* #176 ✅ rn ✅ docs

## 21 Jan 2024

* #174 ✅ rn ✅ docs

## 22 Jan 2024

* #172 ✅ rn ✅ docs
* #177 ✅ rn ✅ docs

## 23 Jan 2024

* #164 ✅ rn ✅ docs
* #179 ✅ rn ✅ docs

### CODE REVIEW

**Review by willmostly — APPROVED** [review]
No comments - LGTM!

File: docs/release-notes.md [hunk 1]
```diff
@@ -18,6 +18,17 @@
 
 # Release notes
 
+## Trino Gateway 5 (?? Jan 2024)
+
+[gateway-ha-5-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/5/gateway-ha-5-jar-with-dependencies.jar)
+
+* Add support for OAuth 2 audience use.  (https://github.com/trinodb/trino-gateway/pull/164)
```
Comment (colebow) [inline docs/release-notes.md:25]: Maybe link to the OAuth docs for what audience use is?

File: docs/release-notes.md [hunk 2]
```diff
@@ -18,6 +18,17 @@
...
+* Add quickstart scripts and documentation. (https://github.com/trinodb/trino-gateway/pull/110)
+* Add project logo. (https://github.com/trinodb/trino-gateway/pull/111)
+* Prevent ignoring of HTTP header configuration. (https://github.com/trinodb/trino-gateway/issue/100)
```
Comment (colebow) [inline docs/release-notes.md:28]:
```suggestion
* Prevent ignoring HTTP header configuration. (https://github.com/trinodb/trino-gateway/issue/100)
```

**Review by colebow — APPROVED** [review]

File: docs/release-notes.md (hunk 1 above)
Comment (mosabua) [reply docs/release-notes.md:25]: We dont have that in the Trino Gateway docs and its too specific in other docs.

File: docs/release-notes.md [hunk 3]
```diff
@@ -18,6 +18,17 @@
 
 # Release notes
 
+## Trino Gateway 5 (24 Jan 2024)
+
+[gateway-ha-5-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/5/gateway-ha-5-jar-with-dependencies.jar)
+
+* Add support for OAuth 2 audience use. (https://github.com/trinodb/trino-gateway/pull/164)
+* Add quickstart scripts and documentation. (https://github.com/trinodb/trino-gateway/pull/110)
+* Add project logo. (https://github.com/trinodb/trino-gateway/pull/111)
```
Comment (martint) [inline docs/release-notes.md:27]: Why are these in the release notes?

**Review by Chaho12 — APPROVED** [review]
LGTM 👍

File: docs/release-notes.md (hunk 3 above)
Comment (mosabua) [reply docs/release-notes.md:27]: Its a significant step for the project

**Pull request merged**